### ISSUE TITLE
failure to automatically generate alt text and image tags when image scaling is removed

### ISSUE THREAD

**Issue by mbateam** [10up/classifai#354]
### Describe the bug

I want to prevent Wordpress from scaling images that are uploaded and added this code as a function:  add_filter( 'big_image_size_threshold', '__return_false' );
once this was done the plugin failed to create text for the image. 

### Steps to Reproduce

add add_filter( 'big_image_size_threshold', '__return_false' ); to the functions then upload an image to the media gallery

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dkotter** [comment]
I was not able to reproduce this in my testing today so wondering if this has since been fixed? I know it's been a bit @mbateam but wondering if you'd be available to test this out again to see if you're problem is fixed?

**Comment by mbateam** [comment]
sorry I thought this was different issue, i will check it out later and get back to you.
